sellercenter-product-port/src/main/java/com/gfg/sellercenter/product/port/ProductReaderService.java

```diff
@@ -0,0 +1,25 @@
+package com.gfg.sellercenter.product.port;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ProductReaderService {
+    public HashMap<Integer, Product> getProducts(List<Integer> productIds) throws IOException {
+        HashMap<Integer, Product> result = new HashMap<Integer, Product>();
+        ObjectMapper mapper = new ObjectMapper();
+        URL url = new URL("http://localhost:8081/api/products/v1?product_ids=[" + productIds.stream().map(String::valueOf).collect(Collectors.joining(",")) + "]");
```

Comment:
GraphQL URL has to be moved to constructor to not return back to this class

Reply:
It's just a copy-paste for now, we need to rewrite it.